packages/data-context/src/sources/migration/codegen.ts

```diff
@@ -444,8 +444,8 @@ export function reduceConfig (cfg: LegacyCypressConfigJson): ConfigOptions {
   }, { global: {}, e2e: {}, component: {} })
 }
 
-export function getSpecPattern (cfg: LegacyCypressConfigJson, testType: TestingType) {
-  const specPattern = cfg[testType]?.testFiles ?? cfg.testFiles ?? (cfg.projectId ? '**/*.{js,ts,tsx,jsx}' : '**/*.cy.{js,jsx,ts,tsx}')
+export function getSpecPattern (cfg: LegacyCypressConfigJson, testType: TestingType, shouldAddCustomE2eSpecPattern?: boolean) {
```

Comment:
Should we have a default value for `shouldAddCustomE2eSpecPattern`? eg 

```js
export function getSpecPattern (cfg: LegacyCypressConfigJson, testType: TestingType, shouldAddCustomE2eSpecPattern?: boolean = false)
```

Where the default is a bit defensive, so I guess it'd be `false` here?